src/main/java/pro/sky/skyproStreamAPI/handler/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,27 @@
+package pro.sky.skyproStreamAPI.handler;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import pro.sky.skyproStreamAPI.exception.EmployeeNotFoundException;
+
+import pro.sky.skyproStreamAPI.Model.ErrorDTO;
+
+import java.awt.image.PixelGrabber;
+import java.util.Date;
+
+@ControllerAdvice
```

Comment:
```suggestion
@RestControllerAdvice
```
для `@RestController`-ов аннотация должна быть `@RestControllerAdvice`